components/brave_ads/core/internal/common/search_engine/search_engine_url_pattern_constants.cc

```diff
@@ -16,34 +16,34 @@ namespace brave_ads {
 const std::string& GetAmazonUrlPattern() {
   static const base::NoDestructor<std::string> kUrlPattern(base::StrCat(
```

Comment:
As discussed in slack we should consider some solution to get rid of these `NoDestructor` instances eventually in an elegant manner.